lib/src/intl_suggestors/utils.dart

```diff
@@ -142,8 +142,16 @@ String toVariableName(String str) {
   if (str.startsWith("'") && str.endsWith("'")) {
     str = str.substring(1, str.length - 1);
   }
+  // This will ignore the first few interpolated variable
```

Comment:
Rather than doing this here, where it's already been turned into a string, so there's less information, consider doing it where it's called, when we actually have the StringInterpolation AstNode. I think `nameForNode` in intl_message_syntax.dart looks promising. Then you can ask a StringInterpolation for e.g. elements and discard the interpolations.